CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Contributing to the Conjur API for Java
+
+## Table of Contents
+- [Prerequisites](#prerequisites)
+- [Building and Testing](#building-and-testing)
+- [Troubleshooting Steps](#troubleshooting-steps)
+- [Contribution Workflow](#contribution-workflow)
+
+## Prerequisites
+### Java
+To work in this codebase, you will need the latest version of the [Java JDK](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) installed.
+
+### Docker
+This project is container-based and therefore, you will need [Docker](https://hub.docker.com/) in order to make and view your changes.
+
+### Access to Conjur Registry (for contributing to testing ONLY)
+To contribute to testing, you will need both OSS and Enterprise (DAP) images. You shouldn't have a problem pulling down the OSS image, but to pull the Enterprise image, you must be granted access to the Conjur registry.
+
+You will want to get in contact with the Infrastructure team via our [Slack](https://slackin-conjur.herokuapp.com/) to be granted access. If you have been granted access, run: `docker login registry2.itci.conjur.net` 
```

Comment:
we need to update this to `registry.tld`, here and in the repo.

Reply:
Changed in docs and in `docker-compose`